code/game/objects/items/weapons/surgery_tools.dm

```diff
@@ -128,24 +151,60 @@
 
 
 /obj/item/weapon/scalpel/laser
-	heat_production = 0
+	icon_state = "stabslash"
 	damtype = "fire"
 	sharpness_flags = SHARP_TIP | SHARP_BLADE | HOT_EDGE
 	var/cauterymode = 0 //1 = cautery enabled
+	var/obj/item/weapon/scalpel/laser/held
```

Comment:
If this var holds an `/obj/item/weapon/cautery/laser`, why is it cast as the `/obj/item/weapon/scalpel/laser` type?

Reply:
I thought it needed to be the typecast of the item to work. I will change it to just be held.

Reply:
It doesn't *need* to be cast as any specific type in order to work on a basic level. If you wish to access type-specific procs and variables, then it needs to be cast as the type which contains those procs and variables. There's also no reason not to typecast it if it's only ever going to hold one thing.
However, it should be cast as the type of object that will be contained in the var, not the type of object the var is placed on. Hence, `var/obj/item/weapon/cautery/laser/held` rather than `var/obj/item/weapon/scalpel/laser/held`.